bundles/com.espressif.idf.ui/plugin.xml

```diff
@@ -178,7 +178,7 @@
          </command>
          <command
                commandId="com.espressif.idf.ui.command.eclipeguide"
-               icon="icons/ESP-IDF Programming Guide.png"
```

Comment:
Actually, it's intended for 'ESP-IDF Eclipse Plugin Guide'  which will navigate to the idf eclipse GitHub page. 

We will have two menu items:
ESP-IDF Programming Guide
ESP-IDF Eclipse Plugin Guide